codalab/apps/web/models.py

```diff
@@ -1244,6 +1249,13 @@ def sortkey(x):
         return results
 
 
+class CompetitionParticipantManager(models.Manager):
```

Comment:
this appears unused? should be deleted?

Reply:
I think so, can't find this class anywhere else in the code.